internal/pkg/template/templates/environment/partials/cdn-resources.yml

```diff
@@ -44,6 +44,7 @@ UniqueAliasesAction:
   Properties:
     ServiceToken: !GetAtt UniqueJSONValuesFunction.Arn
     Aliases: !Ref Aliases
+    FilterFor: !Ref ALBWorkloads
```

Comment:
Why do we need this?

Reply:
discussed offline - needed to remove `InternalALBWorkloads` from `Aliases`